src/views/startup_submission.hbs

```diff
@@ -5,22 +5,22 @@
 <form class="" action="/registerStartup" method="post">
 
   <fieldset>
-    <label for="startup-name">Startup Name: </label>
+    <label for="startup-name">Startup Name </label>
     <input type="text" name="startup-name" id="startup-name" required>
   </fieldset>
 
   <fieldset>
-    <label for="founder-name">Founder/Co-Founder name: </label>
+    <label for="founder-name">Founder/Co-Founder name </label>
```

Comment:
let's remove co-founder in the meantime.
we should have some functionality here, but we will tackle it later.